spire/config/k8s/server-statefulset.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: spire-server
+  namespace: spire
+  labels:
+    app: spire-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: spire-server
+  serviceName: spire-server
+  template:
+    metadata:
+      namespace: spire
+      labels:
+        app: spire-server
+    spec:
+      serviceAccountName: spire-server
+      containers:
+        - name: spire-server
+          image: ghcr.io/spiffe/spire-server:1.5.1
```

Comment:
```suggestion
          image: ghcr.io/spiffe/spire-server:1.8.2
```

Reply:
We need a newer version of SPIRE to be able to use the CredentialComposer plugin (introduced in 1.6.0). Might as well take the latest.